run_operations.py

```diff
@@ -119,140 +124,146 @@ def command_line_parse(args):
 	logger.info("\t\t--startpysar       : %s\n", inps.startpysar)
 	logger.info("\t\t--startinsarmaps   : %s\n", inps.startinsarmaps)
 	logger.info("\t\t--testsheet	    : %s\n", inps.test_sheet)
-	
 
-	
-###################### Auxiliary Functions #####################
 
+###################### Auxiliary Functions #####################
 
-"""
-    Reads template file for the current dataset and parses out the 'ssaraopt' option before creating command options line options 
-    array for ssara_federated_query.py
-    Parameters: none
-    Returns:    ssara_options: [str], array of command line options to run ssara_federated_query with
-"""
 def create_ssara_options():
-	
-	with open('/nethome/'+user+'/insarlab/OPERATIONS/TEMPLATES/'+dataset+'.template', 'r') as template_file:
+	""" Reads template file for the current dataset and parses out the 'ssaraopt' option before creating command options
+		line options array for ssara_federated_query.py
+
+		Parameters: none
+		Returns:    ssara_options: [str], array of command line options to run ssara_federated_query with
+	"""
+
+	with open('/nethome/' + user + '/insarlab/OPERATIONS/TEMPLATES/' + dataset + '.template', 'r') as template_file:
```

Comment:
This seems to be the same pattern mentioned in [another pull request](https://github.com/geodesymiami/rsmas_isce/pull/21/files#r224965732)

Reply:
That is indeed. I will update it there as well. I wrote this not realizing os.getenv() could grab values for environment variables at runtime. That is used instead now.